CMakePresets.json

```diff
@@ -0,0 +1,86 @@
+{
+    "version"   : 3,
+    "cmakeMinimumRequired" : {
+        "major" : 3,
+        "minor" : 22,
+        "patch" : 0
+    },
+    "configurePresets" : [
+        {
+            "name"          : "default",
+            "binaryDir"     : "${sourceDir}/build-with-${presetName}",
+            "cacheVariables" : {
+                "CMAKE_EXPORT_COMPILE_COMMANDS"     : {"type" : "BOOL"  , "value" : "ON"},
+                "CMAKE_BUILD_TYPE"                  : {"type" : "STRING", "value" : "Release"},
+                "CMAKE_CXX_STANDARD"                : {"type" : "STRING", "value" : "17"},
+                "KokkosTools_REUSE_KOKKOS_COMPILER" : {"type" : "BOOL"  , "value" : "ON"},
+                "KokkosTools_ENABLE_EXAMPLES"       : {"type" : "BOOL"  , "value" : "ON"},
+                "KokkosTools_ENABLE_SINGLE"         : {"type" : "BOOL"  , "value" : "ON"},
+                "KokkosTools_ENABLE_MPI"            : {"type" : "BOOL"  , "value" : "ON"},
+                "Kokkos_ROOT"                       : {"type" : "STRING", "value" : "$env{Kokkos_ROOT}"}
```

Comment:
```suggestion
```
This environment variable is used in the normal search procedure, why would we do that?

Reply:
Indeed, weird that I also needed to repeat `Kokkos_ROOT` in the cache variables. Let's remove it and see :)